src/bootstrap/doc.rs

```diff
@@ -263,9 +258,7 @@ fn invoke_rustdoc(builder: &Builder, target: Interned<String>, markdown: &str) {
         t!(t!(File::create(&version_info)).write_all(info.as_bytes()));
     }
 
-    let mut cmd = Command::new(&rustdoc);
-
-    builder.add_rustc_lib_path(compiler, &mut cmd);
+    let mut cmd = builder.rustdoc_cmd(builder.compiler(0, build.build));
```

Comment:
Yeah let's change this to threading through a stage and/or compiler to here. Otherwise I think that `./x.py doc` will compile rustdoc twice, once in the last stage and once in stage0?